content/communityExtensions/dynamicTerrains.md

```diff
@@ -10,26 +10,25 @@ video-content:
 
 The Dynamic Terrain is an extension that provides a way to display a large terrain dynamically morphed from a map of 3D data.
 
-![dynamicTerrain](/img/extensions/dynamicTexture.jpg)
-
-The dynamic terrain is basically a standard BJS mesh, actually a ribbon. It's linked to a camera and moves with it along the World X and Z axes. It's passed a set of geographic data what are each simply 3D coordinates _(x, y, z)_ in the World. This set of data is called the map. According to the current camera position in the World, the dynamic terrain morphs to depict the map at this location.
+The dynamic terrain is a standard BJS mesh, specifically a ribbon. It's linked to a camera and moves with it along the World X and Z axes. It's passed a set of 3D coordinates _(x, y, z)_ in the World. This set of data is called the map. According to the current camera position in the World, the dynamic terrain morphs to depict the map at this location.
 
 ## Installation
 
-Just download the javascript file `dynamicTerrain.js` (or, recommended, the minified version `dynamicTerrain.min.js`) from theBabylon.js [extension repository](https://github.com/BabylonJS/Extensions) folder `DynamicTerrain/dist` : https://github.com/BabylonJS/Extensions/tree/master/DynamicTerrain/dist
+Just download the JavaScript file `dynamicTerrain.js` (or, recommended, the minified version `dynamicTerrain.min.js`) from the Babylon.js [extension repository](https://github.com/BabylonJS/Extensions) folder `DynamicTerrain/dist`: https://github.com/BabylonJS/Extensions/tree/master/DynamicTerrain/dist
 
-Then in your code, declare this script in a html tag **after** the script tag declaring Babylon.js :
+Then in your code, declare this script in an html tag **after** the script tag declaring Babylon.js:
 
 ```html
 <script src="babylon.js"></script>
 <script src="dynamicTerrain.min.js"></script>
 ```
 
+It has also been converted to an [ES6 compatible TypeScript module](https://gist.github.com/danielstenson/07bf127136cd7730dfcfedb4e97a9649).

```

Comment:
I will appreciate the code in the docs and not in a gist of an account we don't have control of (i know it is yours :-) )

Reply:
I see. sorry, i thought you were the author. Let's keep the URL then.